.eslintrc.json

```diff
@@ -5,6 +5,7 @@
     "@zendeskgarden/eslint-config/plugins/react",
     "prettier"
   ],
+  "ignorePatterns": ["node_modules/", "demo/"],
```

Comment:
That's a remnant of the previous CLI generation, will remove

Reply:
Ignoring those files drastically improve the runtime duration of `yarn lint`

Reply:
Removed in https://github.com/zendeskgarden/svg-icons/pull/264/commits/29577628b7d7c0757b2539c30c92b625b94feec7

Reply:
I used the [example from the Gulp doc](https://gulpjs.com/docs/en/getting-started/using-plugins#inline-plugins) as example. I think it's good to leave it as a safety net to avoid the generation failing if there's a non buffer file going through